user-guide/Advanced_Modules/User Defined APIs/Tutorial/UD_APIs_Hello_world_tutorial.md

```diff
@@ -0,0 +1,142 @@
+---
+uid: UD_APIs_Hello_world_tutorial
+---
+
+# Hello world tutorial
+
+This tutorial lesson demonstrates how to create your first custom API. We will create a simple API that, upon execution, will return 'Hello world'.
```

Comment:
Maybe add a note about the prerequisites. I initially added a comment about adding them, but then saw them bein available on the main 'tutorials' page which I did not click on. That page could be missed.